### DISCUSSION TITLE
Release timeline (NET8) question

### DISCUSSION THREAD

**Discussion by msft-mw** [danielpalme/ReportGenerator#634] [Q&A] [answered]
Hello! I saw there was a pending PR (#631) for NET8 support and wanted to ask if there was a timeline for when that would get merged and released. I assume at least not before the full (non-RC) release of NET8 but wasn't sure if that kind of information is known and shareble.

**Comment by danielpalme** [comment, marked as answer]
I will publish a new release as soon as possible. Probably within the next 24 hours.

**Comment by msft-mw** [comment]
Thank you for the update, it's very much appreciated!
